src/main/java/com/artipie/YamlAuth.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie;
+
+import com.amihaiemil.eoyaml.YamlMapping;
+import com.artipie.http.auth.Authentication;
+import java.util.Optional;
+import org.apache.commons.lang3.NotImplementedException;
+
+/**
+ * Authentication implementation based on yaml file with credentials.
+ * @since 0.3
+ * @todo #146:30min Implements this class to find user by credentials and enable test. For more
+ *  details check #146 and note that password is Base64 encoded.
```

Comment:
@olenagerasimova why base64? Standard tools such as `sha256sum` uses plain hex encoding for sha256

Reply:
@g4s8 according to java doc of `com.artipie.http.auth.Authentication#user` param `password` is Base64 encoded. Or I've got something wrong? If it's not encoded, I should correct the test

Reply:
@olenagerasimova the javadoc of `com.artipie.http.auth.Authentication#user` is wrong - I just tested that it's working correctly with plain text passwords (when using http-basic)